aequilibrae/paths/traffic_assignment.py

```diff
@@ -481,14 +481,19 @@ def info(self) -> dict:
             *info* (:obj:`dict`): Pandas dataframe with all the assignment results indexed on link_id
         """
 
-        classes = {}
+        classes = {'matrix_totals': {}}
         for cls in self.classes:
             if len(cls.matrix.view_names) == 1:
-                classes[cls.graph.mode] = {nm: np.sum(cls.matrix.matrix_view[:, :]) for nm in cls.matrix.view_names}
+                classes['matrix_totals'][cls.graph.mode] = {nm: np.sum(cls.matrix.matrix_view[:, :]) for nm in
```

Comment:
This suffers from the same issue we had inside the liner_approximation class.  I'll address it before we merge it